integration-tests/api-gateway/endpoints/index.ts

```diff
@@ -0,0 +1,109 @@
+import { getSSMParameter } from "../../step-functions/aws/resources/ssm-param-helper";
+import {
+  Payload,
+  getJarAuthorizationPayload,
+} from "../aws/crypto/create-jar-request-payload";
+import { claimSet, PRIVATE_API } from "../aws/env-variables";
+
+let publicEncryptionKeyBase64: string;
+let privateSigningKey: any;
+
+const environment = process.env.Environment || "dev";
```

Comment:
you can remove this use the one in constant file and default to `localdev`

Reply:
audience and authorizationEndpoint in payload both use dev in the URL instead of localdev so this cannot be removed